### PR TITLE
Strips out images from the repo and moves them to a CDN.

### LINKED ISSUE #280: Point images to new CDN
As a developer
So I don't have huge image files to download whenever I update Bower
I want to extract the images out of the repo and into the makers-assets repo

### PR DESCRIPTION
**Pull request by pitchinvasion**
In order to do this I created a new image_path helper which points to the location of the CDN.

I also had to parse from Github the partner logo images which turned out to be surpisingly simple to do.

If this is OK with everyone then I'll merge it in and then update bower and all of the micro sites to point to the new CDN.

Note this is a quick fix at some point we should switch them to a proper CDN but it'll do for now

Closes #280

### CODE REVIEW

**Comment by sdorunga1** [comment]
Looks good.

Still not sure what assets.makers academy.com is :)

File: Gemfile.lock [hunk 1]
```diff
@@ -172,10 +161,13 @@ PLATFORMS
...
   middleman-livereload (~> 3.1.0)
   middleman-meta-tags
   middleman-search_engine_sitemap
   neat (~> 1.7.2)
+  pry
   rspec
   sass
+
+BUNDLED WITH
+   1.10.5
```
Comment (spike01) [inline Gemfile.lock]: This? Again?

Comment (spike01) [inline Gemfile.lock]: Or are we all on the same Bundler now?

**Comment by spike01** [comment]
It's the "Makers CDN" (i.e. using GitHub pages (i.e. using Fastly))

:+1: on this assuming it works

File: Gemfile.lock (hunk 1 above)
Comment (pitchinvasion) [inline Gemfile.lock]: it should be the last time if we're all on bundler 1.10.5

**Pull request merged**